contrib/clojure-package/examples/pre-trained-models/src/pre_trained_models/predict_image.clj

```diff
@@ -40,21 +40,12 @@
     (io/copy in out)))
 
 (defn get-image [url show?]
-  (let [fname "test-image.jpg"
-        _ (download url fname)
-        image (-> (img/load-image fname)
-                  (img/resize h w))
-        pixels (img/get-pixels image)
-        rgb-pixels (reduce (fn [result pixel]
-                             (let [[rs gs bs] result
-                                   [r g b _] (pixel/unpack-pixel pixel)]
-                               [(conj rs r) (conj gs g) (conj bs b)]))
-                           [[] [] []]
-                           pixels)]
-    (when show? (img/show image))
```

Comment:
Is there a way to show the image to the user if the `show?` flag is true? `imshow`?